README.md

```diff
@@ -153,6 +153,6 @@ in your current shell. But it won't be available next time you log in.
 To install it, add the line above to your `.bashrc`.
 
 [1]: https://github.com/endlessm/flapjack/blob/master/example.flapjack.ini
-[2]: http://www.youtube.com/watch?v=70Kl9ft5DGA&t=40m4s
+[2]: http://www.youtube.com/watch?v=hE5nJrGnX-U&t=40m4s "GUADEC 2017 - Tristan Van Berkom - GNOME Build Strategies and BuildStream"
```

Comment:
While here can you switch this to https://? YouTube will just redirect you, but you might as well cut to the chase.